models/article-models.js

```diff
@@ -14,14 +14,43 @@ exports.articleById = (article_id) => {
     });
 };
 
-exports.returnArticles = () => {
-  return db
-    .query(
-      `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC`
-    )
-    .then(({ rows }) => {
-      return rows;
-    });
+exports.returnArticles = (query) => {
+  const { sort_by, order } = query;
```

Comment:
As mentioned above, I would prefer to see this done in the controller, that way I can see what I am passing down and what I expect here. From the controllers side I have no idea what is being passed down, only when I look at this side can i see this, which is not ideal.